LocalFeeder/dss_functions.py

```diff
@@ -70,12 +74,27 @@ def get_pvsystems(dss):
         datum["numPhase"] = NumPhase
         datum["numPhases"] = NumPhase
         datum["power"] = dss.CktElement.Powers()[: 2 * NumPhase]
+        datum["node_names"] = get_all_nodes(buses)
 
         data.append(datum)
         PV_flag = dss.PVsystems.Next()
     return data
 
 
+def get_all_nodes(buses: list[str]):
+    """Get all nodes from list of buses."""
+    all_nodes = []
+    for bus in buses:
+        sub_bus = bus.split(".")
+        core_name = sub_bus[0].upper()
```

Comment:
Why is upper() being called here? Won't that potentially cause naming conflicts with other parts of the code?

Reply:
YNodeOrder() uses the upper case for bus names. So even in our test_data/master.dss it has the upper case instead.